substrate/frame/election-provider-multi-block/src/mock/mod.rs

```diff
@@ -464,11 +464,20 @@ pub trait ExecuteWithSanityChecks {
 
 impl ExecuteWithSanityChecks for sp_io::TestExternalities {
 	fn execute_with_sanity_checks(&mut self, test: impl FnOnce() -> ()) {
+		self.execute_with(all_pallets_integrity_test);
```

Comment:
might be good to note why we do this -- because `construct_runtime` generates a unit test that checks them all, but we
may tweak things like `type SignedValidationPhase` per-test, so we re-run these again.